engines/director/lingo/lingo.h

```diff
@@ -443,6 +443,9 @@ class Lingo {
 
 	Common::String _floatPrecisionFormat;
 
+	int _selStart;
+	int _selEnd;
```

Comment:
These properties should probably belong to Movie, not Lingo, to avoid conflicts with movies in other windows.

Reply:
thanks